### ISSUE TITLE
Inception: Training with Imagenet

### ISSUE THREAD

**Issue by redserpent7** [tensorflow/models#907]
Hi,

I downloaded the imagenet images manually from the URLs list and I have them all now. I am trying to train the inception model from scratch but I am not sure where to start.

I took a copy of the download_and_preprocess scripts and modified them to skip the download however I am puzzled since the script suggests that there should be two sets of images; one for training and the other for evaluation. I am not sure how to split the images.

Is there a list somewhere that I can get the images required for evaluation? I cannot seem to find this information on the ImageNet site.

**Comment by asimshankar** [comment]
@redserpent7 : This question is probably better asked on [stackoverflow](http://stackoverflow.com/questions/tagged/tensorflow) as we aim to keep the github issues focused on bugs and feature requests instead of usage support. Hence, closing this issue.

Most likely, your modifications caused you to skip the download of the validation set
(see https://github.com/tensorflow/models/blob/master/inception/inception/data/download_imagenet.sh#L74)
